install.lua

```diff
@@ -31,12 +31,12 @@ end
 
 local function rewriteDofile(filename, required)
```

Comment:
no we could def just blind replace them... what would be the gsub command for that?